packages/react/src/themes/teams/index.ts

```diff
@@ -21,22 +21,34 @@ const declareSvg = (svgIcon: SvgIconSpec): ThemeIconSpec => ({
 
 const declareFontBased = (fontIcon: FontIconSpec): ThemeIconSpec => ({ icon: fontIcon })
 
-const icons: ThemeIcons = Object.keys(svgIconsAndStyles as {
+const getIcon = iconAndMaybeStyles => {
```

Comment:
this logic seems to be duplicated - lets extract it and consume from the single place